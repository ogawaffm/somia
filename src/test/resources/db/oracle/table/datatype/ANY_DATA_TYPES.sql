BEGIN
    EXECUTE IMMEDIATE 'DROP TABLE TEST.ANY_DATA_TYPES';
EXCEPTION
    WHEN OTHERS THEN
        IF SQLCODE != -942 THEN
            RAISE;
        END IF;
END;
/
CREATE TABLE TEST.ANY_DATA_TYPES
(
    ANYTYPE_COL    ANYTYPE,
    ANYDATASET_COL ANYDATASET

)