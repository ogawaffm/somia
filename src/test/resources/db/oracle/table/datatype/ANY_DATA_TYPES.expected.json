{
  "name": "ANY_DATA_TYPES",
  "catalogName": "",
  "schemaName": "TEST",
  "typeName": "TABLE",
  "comment": "",
  "columns": [
    {
      "name": "ANYTYPE_COL",
      "type": {
        "name": "ANYTYPE",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 3752,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "ANYDATASET_COL",
      "type": {
        "name": "ANYDATASET",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 3752,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    }
  ],
  "indexes": [
    {
      "name": "",
      "columns": [],
      "isUnique": true,
      "comment": "",
      "qualifierName": "",
      "isClustered": false,
      "isHashed": false,
      "isStatistic": true,
      "whereClause": "",
      "cardinality": 0,
      "pages": 0
    }
  ],
  "collation": "",
  "primaryKey": null,
  "bestRowIdentifier": null
}